leesy/style/portfolio.css

```diff
@@ -0,0 +1,145 @@
+html, body {
+	margin: 0;
+	padding: 0;
+	height: 100%;
+	width:200%;
+	color: #033C6B;
+
+}
+
+
+header {
+	position: fixed;
+	left: 65px;
+	z-index: 999;
+	text-align: right;
+
+
+}
+
+.nav_container {
+	line-height: 60px;
+	padding-left: : 1%;
+	position: fixed;
+	white-space: nowrap;
+	z-index: 12;
+}
+
+
+
+
+.page_link {
+	display: inline-block;
+	margin: 0 40px 0 0;
+	font-style: oblique;
+
+}
+
+.nav_container a{
+	color: green;
+	padding: 20px 0 0 0;
+	text-decoration: none;
+}
+
+.page_link a {
+	color: #0076D6;
+	border-bottom: 0px solid #cccccc;
+}
+
+a:hover{
+	color: #0000FF;
+	border-bottom: 10px solid #0000FF;
+}
+
+a:hover {
+	border-bottom: 10px solid #0000FF;
+	color: #0000FF;
+	font-weight: bold;
+}
+
+
+
+#swipe{
+	display: inline-block;
+	padding: 20px 0 0 65vw;
+	font-style: oblique;
+	color: #033C6B;
+}
+
+
+
+#slide {
+	height:100%;
+	width:25%;
+	max-height:100%;
+	float:left;
+	box-sizing: border-box;
+}
+
+
+
+#slide img{
+	position: relative;
+
+	text-align: center;
+	margin: auto;
+	top: 20%;
+	left: 20%;
+	bottom: 20%;
+	right:20%;
+}
+
+
+
+.slide1 {
+
+	margin-left: 2%;
+}
+
+/*
+.slide2, .slide3, .slide4 {
+display: inline-block;
+position: relative;
+text-align: center;
+padding-top: 13vh;
+}
+
+.slide2 > .inner{
+
+background:#4380FF;
+
+
+
+background:#FAC8FA;
+}
+.slide4 > .inner{
+margin: 0 auto;
+width: 60vw;
+height: 65vh;
+position: relative;
+background:#FF9999;
+}*/
+
+
+#text-container {
+	margin-top: 20vh;
+	text-align: center;
+}
+
+#text-container a{
+
+	text-decoration: none;
+	color: #033C6B;
+	font: helvetica;
+}
+
+#text-container a:hover{
+color: #0000FF;
+border-bottom: 10px solid #0000FF;
+}
+
+#text-container a:hover {
+border-bottom: 10px solid #0000FF;
+color: #0000FF;
+font-weight: bold;
+}
```

Comment:
Commenting here, but it applies to all of your css files. A good way to DRY up your code might be to only have one stylesheet that applies to all of your pages.